3p/ampanalytics-lib.js

```diff
@@ -0,0 +1,264 @@
+/**
```

Comment:
this name is internal in our repo only, so it's a good idea to keep it consistent with other naming.
we can definitely use a different name for the compiled library (maybe including ampanalytics in the name).